From 0066ff9bd83670d5761394948c6bcd4d1080eb97 Mon Sep 17 00:00:00 2001
From: Jiri Olsa <jolsa@redhat.com>
Date: Tue, 19 Aug 2014 15:27:30 -0400
Subject: [tools] perf: Remove unneeded include

Message-id: <1408462094-14194-282-git-send-email-jolsa@redhat.com>
Patchwork-id: 88219
O-Subject: [PATCH RHEL7.1 BZ1131394 281/325] perf tools: Remove unneeded include
Bugzilla: 1131394
RH-Acked-by: Prarit Bhargava <prarit@redhat.com>
RH-Acked-by: Tony Camuso <tcamuso@redhat.com>

From: Jiri Olsa <jolsa@kernel.org>

Bugzilla: 1131394
https://bugzilla.redhat.com/show_bug.cgi?id=1131394

upstream
========
commit 8ce000e83848578a621d64eccdc88bd34c2fc70c
Author: Rodrigo Campos <rodrigo@sdfg.com.ar>
Date: Wed Nov 6 22:20:54 2013 +0000

description
===========
There is no point in sort.h including itself.

The include was added when the file was created, in commit "perf tools:
Create util/sort.and use it" (dd68ada2d) and added a include to "sort.h"
in lot of files (all the files that started using the file). It was
probably added by mistake on sort.h too.
---

Signed-off-by: Jarod Wilson <jarod@redhat.com>

diff --git a/tools/perf/util/sort.h b/tools/perf/util/sort.h
index f4cc147..43e5ff4 100644
--- a/tools/perf/util/sort.h
+++ b/tools/perf/util/sort.h
@@ -22,7 +22,6 @@
 #include "parse-events.h"
 
 #include "thread.h"
-#include "sort.h"
 
 extern regex_t parent_regex;
 extern const char *sort_order;
-- 
1.7.1